UI/window-basic-main.cpp

```diff
@@ -1674,12 +1683,14 @@ void OBSBasic::InitBasicConfigDefaults2()
 					      "Pre23Defaults");
 	bool useNV = EncoderAvailable("ffmpeg_nvenc") && !oldEncDefaults;
 
+	bool useX264 = EncoderAvailable("obs_x264");
+	const char *h264_fallback =
+		(useX264 ? SIMPLE_ENCODER_X264 : SIMPLE_ENCODER_OPENH264);
```

Comment:
```suggestion
		useX264 ? SIMPLE_ENCODER_X264 : SIMPLE_ENCODER_OPENH264;
```

Reply:
It's nitpicking, let it unresolved until a another person give his opinion.

Reply:
Huh, TIL that GitHub won't run CI if there are unresolved conversations if "all conversations must be resolved before merging" is set.

Reply:
No, it prevents merging but CI runs are not unlimited, so old runs need to finish before running newer.